src/simplewallet/simplewallet.cpp

```diff
@@ -7034,7 +7034,7 @@ bool simple_wallet::show_transfers(const std::vector<std::string> &args_)
 
     message_writer(color, false) << formatter
       % transfer.block
-      % transfer.direction
+      % tr(transfer.direction.c_str())
```

Comment:
You use tr on literals, since a tool gathers the literal to put it in a file. So tr should be moved to whatever code assigns transfer.direction.

Reply:
Oh ok, didn't know.
If I move the tr to whatever assigns transfer.direction, then the csv will also be using the translated string. Should I just not use tr as all?

Reply:
Ah, good point. Hmm. I suppose if they both have to have the same strings, translated is probably best...